libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.component.html

```diff
@@ -1,6 +1,13 @@
 <div id="emptyDiv" *ngIf="isZero" class="arrow-variation zero" [ngClass]="size"></div>
 <div *ngIf="!isZero" class="arrow" [ngClass]="size">
-  <svg width="100%" height="100%" viewBox="0 0 24 24" style="display: block">
-    <path [attr.fill]="arrowColor" [attr.d]="arrowDirection" />
-  </svg>
+  <ng-template #svgTemplate>
+    <ng-container>
+      <svg width="100%" height="100%" viewBox="0 0 24 24" style="display: block">
+        <path [attr.fill]="arrowColor" [attr.d]="arrowDirection" />
+      </svg>
+    </ng-container>
```

Comment:
The `ng-container` tag inside the `svgTemplate` seems unnecessary. You could write instead:
`<ng-container *ngIf="printing; else svgTemplate">
    <img ...>
</ng-container>
<ng-template #svgTemplate>
    <svg ...></svg>
</ng-template>`